src/PageView/WorkoutsMain/WorkoutExerciseList/WorkoutExerciseList.css

```diff
@@ -13,5 +13,16 @@
 
 .item .detailsReps {
     color: #126195;
+    font-weight: bolder;
 }
 
+.ExercisesList .containerBottom .item {
+    background-color: rgb(243, 240, 240);
```

Comment:
if you're going to use rgb colour, I think you can opt out opacity and change colour to rgba(243, 240, 240, 0.6)